use std::process::ExitCode;

fn main() -> ExitCode {
    ExitCode::FAILURE
}
